testsuite/MDAnalysisTests/topology/test_guessers.py

```diff
@@ -79,6 +79,20 @@ def test_guess_atom_element_singledigit(self):
     def test_guess_atom_element_1H(self):
         assert guessers.guess_atom_element('1H') == 'H'
         assert guessers.guess_atom_element('2H') == 'H'
+    
+    @pytest.mark.parametrize('name, element', (
+        ('AO5*', 'O'),
+        ('F-', 'F'),
+        ('HB1', 'H'),
+        ('OC2', 'O'),
+        ('1he2', 'H'),
+        ('3hg2', 'H'),
```

Comment:
I was going to ask for more elements to be added to the table, but then obviously it will make `he` and `hg` fail.  I guess there's not many people doing MD of mercury....

Reply:
Thanks for the review! (Which I meant to say earlier). I agree that more elements would have been nice -- I originally had helium and calcium in there before realising the obvious ambiguity. As GROMOS force fields have hydrogens called HG in threonine and HE in phenylalanine, I thought element diversity could be left as a problem for a future era of simulation.